content/en/blog/_posts/2021-01-28-PSP-deprecation.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: 'What deprecation of Pod Security Policies means'
+date: 2021-01-28
+slug: psp-deprecation
+---
+
+**Authors:** Matt Broberg (RedHat), Kaslin Fields (Google), Chris Short (RedHat)
+
+Pod Security Policies (PSP), starting with the Kubernetes 1.21, will begin the [process of deprecation](https://kubernetes.io/docs/reference/using-api/deprecation-policy/) with the intention to fully remove it in a future release. This bears the question: what will replace Pod Security Policies in the future? In short, [Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/). For more details, keep reading.
+
+
+## What do Pod Security Policies do now?
+
+> A Pod Security Policy is a cluster-level resource that controls security sensitive aspects of the pod | specification. The [PodSecurityPolicy](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#podsecuritypolicy-v1beta1-policy) object defines a set of conditions that a pod must run with in order to be accepted into the system, as well as defaults for the related fields. They allow an administrator to control a number of security-related functions.
+
+Doc: [https://kubernetes.io/docs/concepts/policy/pod-security-policy/](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
+
+
+## Why are Pod Security Policies being deprecated? 
+
+To watch an explanation of PSP removal, you can check out this session by SIG-Auth at KubeCon NA 2019:
+
+[https://youtu.be/SFtHRmPuhEw?t=953](https://youtu.be/SFtHRmPuhEw?t=953) 
```

Comment:
Could we embed the video? Need to defer to a blog team member on whether this will work. 

```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/SFtHRmPuhEw?start=953" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Reply:
Ignore this -- we should use the Hugo short code instead of iframes.